Makefile

```diff
@@ -5,13 +5,14 @@ PWD := $(shell pwd)
 # Determine OS & Arch for specific OS only tools on Unix based systems
 OS := $(shell uname | tr '[:upper:]' '[:lower:]')
 ifeq ($(OS),darwin)
-	SED := gsed
+	SED ?= gsed
 else
-	SED := sed
+	SED ?= sed
 endif
 
 TOOLS_DIR := $(PWD)/internal/tools
 
+MARKDOWN_LINK_CHECK_ARG= # pass extra arguments such as --exclude '^http'
```

Comment:
would we expect someone to modify the value in the file or could I just pass it in cmd line somehow? (Sorry, I'm a noob when it comes to makefiles)

Reply:
The variable is empty, the part `# ...` is just a comment. So yes, a dev would have to pass a value on the command line.